### PR TITLE
Fix routes reset namespace display

### PR DESCRIPTION
**Pull request by pbishop16**
Resolves: #832 
reset-namespace-demo-2

### CODE REVIEW

File: tests/acceptance/route-tree-test.js [hunk 1]
```diff
@@ -129,13 +143,16 @@ module('Route Tree Tab', function(hooks) {
       "PostRoute",
       "application",
       "application",
+      "comments",
```
Comment (rwwagner90) [inline tests/acceptance/route-tree-test.js]: Do we need `"comments"` twice here?

Comment (pbishop16) [reply tests/acceptance/route-tree-test.js]: @rwwagner90 I double checked and the test seems to want "comments" listed twice similar to "application" and "post".

File: tests/unit/check-current-route-test.js [hunk 2]
```diff
@@ -3,14 +3,152 @@ import { module, test } from 'qunit';
...
+        currentRoute: {
+          url: '/posts/show/comments',
+          name: 'posts.show.comments',
+        },
+        routeValue: {
+          url: '/comments',
+          name: 'comments',
+        },
+        message: 'resource match fails when current route does not match the root route',
+      },
```
Comment (pbishop16) [inline tests/unit/check-current-route-test.js]: @rwwagner90 You are very right. I did run into this edge case. The above test two cases roughly model this scenario.

**Edge case Failure**
example-fail

Adding the following URL check `const hasMatchingURL = routeURL === currentRouteURL;` resolved this edge case.

**Edge Case Success**
example-success

Please let me know if I failed to address your concern.

**Comment by pbishop16** [comment]
@rwwagner90 You are very right. I did run into this edge case. The
